tests/parsers/plist_plugins/macos_background_items.py

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""Tests for the Mac OS background items plist plugin."""
+
+import unittest
+import sys
+
+from plaso.parsers.plist_plugins import macos_background_items
+
+from tests.parsers.plist_plugins import test_lib
+
+
+class MacOSBackgroundItemsPlistPluginTest(test_lib.PlistPluginTestCase):
+  """Tests for the Mac OS background items plist parser plugin."""
+
+  def testProcessWithBackgroundItemsBtm(self):
+    """Tests the Process function with a backgrounditems.btm file."""
+    plist_name = 'backgrounditems.btm'
+
+    if sys.version_info < (3, 8):
```

Comment:
Python 3.7 is EOL (https://devguide.python.org/versions/) I'll remove it from the CI test versions.